### PR TITLE
Update capybara and selenium

### PR DESCRIPTION
**Pull request by Sgtpluck**
### Description of Changes:
Updating our capybara and selenium setup as the gem we were relying on hasn't been updated since 2014

### PR Checklist:

1. [ ] Have you linted and tested your code locally prior to submission?
2. [ ] Have you tagged the appropriate dev(s) for review?
3. [ ] Have you linked to any relevant tickets or conversations?

### PR Review Standards: 
- Consider using [Conventional Comments](https://conventionalcomments.org/) to ensure that your feedback is clear and actionable.
- Ideally, PRs should be reviewed by at least 2 team members.
- All PRs must be approved before being merged.

### CODE REVIEW

File: spec/support/capybara.rb [hunk 1]
```diff
@@ -1,9 +1,9 @@
 require 'capybara/rspec'
-require 'webdrivers/chromedriver'
+
 require 'rack_session_access/capybara'
 
 Capybara.register_driver :headless_chrome do |app|
-  browser_options = Selenium::WebDriver::Chrome::Options.new
+  browser_options = driver = Selenium::WebDriver::Chrome::Options.new
```
Comment (zachmargolis) [inline spec/support/capybara.rb:6]: looks like maybe we don't need the extra `device` assignment?
```suggestion
  browser_options = Selenium::WebDriver::Chrome::Options.new
```

**Review by zachmargolis — APPROVED** [review]
LGTM

File: spec/support/capybara.rb (hunk 1 above)
Comment (Sgtpluck) [reply spec/support/capybara.rb:6]: whoops yeah this was a typo, thanks

**Comment by Sgtpluck** [comment]
Closing, as this was fixed in https://github.com/18F/identity-dashboard/pull/707

**Pull request closed without merging**